src/Hosting/Hosting/src/Host.cs

```diff
@@ -90,10 +92,26 @@ public static IHostBuilder CreateDefaultBuilder(string[] args)
             })
             .ConfigureLogging((hostingContext, logging) =>
             {
+                var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+
+                // IMPORTANT: This needs to be added *before* configuration is loaded, this lets
+                // the defaults be overridden by the configuration.
+                if (isWindows)
+                {
+                    // Default the EventLogLoggerProvider to warning or above
+                    logging.AddFilter<EventLogLoggerProvider>(level => level >= LogLevel.Warning);
```

Comment:
This overrides any of the general category logging levels and has to be specifically overriden using an `"EventLog"` sub-setting. Is this desirable?

Reply:
e.g.
```
  "Logging": {
    "LogLevel": {
      "Microsoft.AspNetCore.Hosting.Diagnostics": "None"
    }
  },
```
Will not override it

Reply:
Is this documented anywhere? Had a lot of trouble figuring this out.